test/EndToEnd/tests/GetPackageTest.ps1

```diff
@@ -181,6 +181,8 @@ function Test-GetPackageForProjectReturnsCorrectPackages2 {
 }
 
 function Test-GetPackageForFSharpProjectReturnsCorrectPackages {
+    [SkipTest('https://github.com/NuGet/Home/issues/11291')]
+    param()
```

Comment:
It's better to do this in a separate PR beause
1. It can get merged faster, as it won't get slowed down by other comments in this PR
2. If this PR introduces problems into the product, or build, it can be reverted, without re-enabling a flaky test.